src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(10, 2)).toStrictEqual([5, 5]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(5, 1)).toStrictEqual([5]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
```

Comment:
Please add a test case description in the test function as a string to make it more readable and understandable. For example: `test('should sort parts ascending if they are not equal', () => {...})`